Refactor scripts for ease of unittesting the db dump recent script

gregcorbett opened the pull request:
~~Depends on #23~~

- Ensures all timestamps are treated as UTC
- Moves existing functionality of check_db_dump_recent.py into a class for ease of unit testings
- Updates the README.md

Sae126V commented on check/check_db_dump_recent.py:
```diff
@@ -0,0 +1,83 @@
+"""
+This file contains a script to check the GOCDB failover process is happening.
+
+Specifically, it checks the log file passed as the first arguement for evidence
```
Seems there is a typo for arguement -> argument and suceeded -> succeeded.

gregcorbett replied:
Fixed in [db4a211](https://github.com/GOCDB/gocdb-failover-scripts/pull/22/commits/db4a211a3c4c5ee479bf47236b305dcb85c342af)

Sae126V commented on importDBdmpFile/1_runDbUpdate.sh:
```diff
@@ -1,30 +1,30 @@
-# local oracle xe instance using sqlplus. 
-# Notes: 
-# - The updatelog.txt file is appended each time with log messages whenever the script is ran. 
+# Script for downloading the gocdb5 db dmp file and importing into the
```
I see that there is a PR #23  addressing whitespaces. Is it being repeated here ?

Sae126V commented:
It seems there is an empty file: check/__init__.py. Is it intentional ?

Sae126V commented:
I finished my review. LGTM

gregcorbett commented:
> It seems there is an empty file: check/**init**.py. Is it intentional ?

Yes, it is used to mark a directory as a Python package so that unit tests will be able to import it.

gregcorbett commented on check/check_db_dump_recent.py:
```diff
@@ -0,0 +1,83 @@
+                return RETURN_CODE_CRITICAL
+
+            print("The failover process last suceeded at %s" % last_success)
+            # If the failover process hasn't suceeded in a while, the
```
fixed in [e0b8f84](https://github.com/GOCDB/gocdb-failover-scripts/pull/22/commits/e0b8f84d7314bea38ae801e0de4ea9664d489e51)

gregcorbett commented:
I have rebased on top of the latest `main` (which has removed the whitespace commit already applied in #21) and squashed the SPG fixes into one commit.
